test/Ocelot.UnitTests/Configuration/SecurityOptionsCreatorTests.cs

```diff
@@ -31,20 +52,57 @@ public void should_create_security_config()
 
             var expected = new SecurityOptions(ipAllowedList, ipBlockedList);
 
-            this.Given(x => x.GivenThe(fileRoute))
-              .When(x => x.WhenICreate())
-              .Then(x => x.ThenTheResultIs(expected))
-              .BDDfy();
+            this.Given(x => x.GivenThe(globalConfig))
+                .Given(x => x.GivenThe(new FileSecurityOptions()))
+                .When(x => x.WhenICreate())
+                .Then(x => x.ThenTheResultIs(expected))
+                .BDDfy();
+        }
+
+        [Fact]
+        public void should_create_global_route_security_config()

```

Comment:
No traits!

Reply:
I fixed putting fist char uppecare likes the others tests